experimental/skills/DialogToDialog/DialogRootBot/Controllers/SkillController.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+using Microsoft.Bot.Builder.Skills;
+
+namespace Microsoft.BotBuilderSamples.DialogRootBot.Controllers
+{
+    /// <summary>
+    /// A controller that handles Skill replies to the bot.
+    /// This example uses <see cref="SkillHandler"/> that is registered as <see cref="ChannelServiceHandler"/> in startup.cs.
```

Comment:
```suggestion
    /// This example uses the <see cref="SkillHandler"/> that is registered as a <see cref="ChannelServiceHandler"/> in startup.cs.
```

Reply:
Updated comment in code

---
In reply to: [356211240](https://github.com/microsoft/BotBuilder-Samples/pull/2032#discussion_r356211240) [](ancestors = 356211240)